client/webserver/locales/pl-pl.go

```diff
@@ -177,6 +177,7 @@ var PlPL = map[string]string{
 	"how_reg":                     "Jak chcesz uiścić opłatę rejestracyjną?",
 	"All markets at":              "Wszystkie rynki na",
 	"pick a different asset":      "wyierz inne aktywo",
```

Comment:
Dang it! Found a typo. It should be  "wybierz inne aktywo", Sorry!
```suggestion
	"pick a different asset":      "wybierz inne aktywo",
```